### PR TITLE
TinyMCE typings (WIP)

### PR DESCRIPTION
**Pull request by martinduparc**
This is a WIP. Please contribute

### CODE REVIEW

File: tinymce/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline tinymce/tsconfig.json]: Make this `true`, and add `"noImplicitThis": true` above it.

File: tinymce/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// Type definitions for TinyMCE (WIP)
+// Project: https://github.com/tinymce/tinymce
+// Definitions by: Martin Duparc <https://github.com/martinduparc/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface TinyMceObservable {
```
Comment (andy-ms) [inline tinymce/index.d.ts]: These could all go in a `TinyMce` namespace instead of prefixing the names.

**Review by ghost — COMMENTED** [review]
Linter will tell you the rest.
I doubt you'll get anyone to contribute to this just by making a pull request. But if you find these types usable in your own code then we could publish these as-is.

File: tinymce/tsconfig.json (hunk 1 above)
Comment (martinduparc) [inline tinymce/tsconfig.json]: Will do

**Comment by martinduparc** [comment]
Hi @andy-ms 

I wanted to get things started and will be adding to the file. It does the job for now.

Thanks

**Pull request merged**